lib/inspec/rspec_json_formatter.rb

```diff
@@ -52,7 +52,12 @@ def stop(notification)
       format_example(example).tap do |hash|
         e = example.exception
         next unless e
-        hash[:message] = exception_message(e)
+
+        if example.metadata[:sensitive]
+          hash[:message] = '*** sensitive ***'
```

Comment:
Can I suggest we change this to `*** sensitive output suppressed ***` to make it extra clear?